README.md

```diff
@@ -113,19 +113,15 @@ VS Code users will also see a ruler at the 100 character mark, helping the devel
 
 Links to the homepages of libraries used in this project, listed here for easy reference.
 
-- [Material-UI](https://next.material-ui.com/)
+- [MUI](https://mui.com/)
 
-  Material-UI is a widely-used React implementation of Google's Material Design. It provides a comprehensive toolkit of interface components, along with a set of usability guidelines and best practices for using them.
+  MUI is a widely-used React component library originally based on Google's Material Design standards. It provides a comprehensive toolkit of interface components, along with a set of usability guidelines and best practices for using them.
```

Comment:
Shouldn't we say "Material-UI" somewhere in here so that people know what MUI stands for?

Reply:
What's the reason for removing it entirely?

Reply:
Actually, they rebranded as of v5, and it no longer stands for `Material-UI`, the package/company is just `MUI` now. They say it stands for `**M**aterials to build **UI**s` now, but the npm package, the website, and the official name are just `MUI` now. Check out [their blog post about it](https://mui.com/blog/material-ui-is-now-mui/) for more details

Reply:
I figured that, but I still think that we should put "Materials to Build UIs" in the documentation

Reply:
I disagree just for discoverability reasons. There is no content anywhere under the name 'Materials to Build UIs', they don't even use that name themselves. I think the only occurrence of those words might be in that one blog post. MUI is the name of the package/company.

Reply:
![image](https://user-images.githubusercontent.com/35319956/135679250-e27f4090-eed8-4a7e-ae1c-7243ae6041b1.png)

![image](https://user-images.githubusercontent.com/35319956/135679738-f2630b91-fb15-4cbb-851b-a00fe95194bc.png)

![image](https://user-images.githubusercontent.com/35319956/135679288-ef555503-1b2b-40e7-802b-41c3ebdeb3ad.png)

![image](https://user-images.githubusercontent.com/35319956/135679325-39c08ce1-3c49-46cd-9cb9-11be086f2d8d.png)

![image](https://user-images.githubusercontent.com/35319956/135679373-f02fc740-317f-4913-afab-3a9e2281300b.png)

Their GitHub profile, Twitter profile, and website all use MUI ubiquitously as their name, and don't reference that tag line even once. Notice how they're telling people to replace `Material-UI` with `MUI` when referring to them.

Reply:
You are telling me nothing, you have said this already. I clearly do not care what MUI has branded themselves as. I am saying that it would be more clear for our users if we gave the acronym a name because "MUI" means absolutely nothing to anyone who is first joining the team and is exploring our documentation. I don't think they should have to go to the website and explore it just to see what the actual name stands for.

Reply:
I don't see how `Material-UI` was any more descriptive than `MUI`. And they don't have to click through to the website because the next words after `MUI` are a much more informative definition than `Materials to Build UIs`. The point is that `MUI` is no longer an acronym, it's just a name.

Reply:
I recognize that. I still think we should put the old name or the more descriptive name in there. It references what it used to be called if nothing else.